quickevent/app/plugins/Runs/src/eventstatisticswidget.cpp

```diff
@@ -171,30 +171,31 @@ QVariant EventStatisticsModel::value(int row_ix, int column_ix) const
 	if(column_ix == col_timeToCloseMs) {
 		int stage_start_msec = eventPlugin()->stageStartMsec(eventPlugin()->currentStageId());
 		int curr_time_msec = QTime::currentTime().msecsSinceStartOfDay();
-		//int runners_count = value(row_ix, col_runnersCount).toInt();
+		int runners_count = value(row_ix, col_runnersCount).toInt();
 		int runners_finished = value(row_ix, col_runnersFinished).toInt();
-		if(runners_finished == 0) {
-			return QVariant();
+		if(runners_count == runners_finished) {
+			return 0;
 		}
-		else {
-			int start_last_msec = value(row_ix, col_startLastMs).toInt();
-			int time3_msec = value(row_ix, col_time3Ms).toInt();
-			int time_to_close_msec = stage_start_msec + start_last_msec + time3_msec - curr_time_msec;
-			if(time_to_close_msec < 0)
-				time_to_close_msec = 0;
-			return time_to_close_msec;
+		if(runners_finished < 3) {
+			return QVariant();
 		}
+		int start_last_msec = value(row_ix, col_startLastMs).toInt();
+		int time3_msec = value(row_ix, col_time3Ms).toInt();
+		int time_to_close_msec = stage_start_msec + start_last_msec + time3_msec - curr_time_msec;
+		if(time_to_close_msec < 0)
+			time_to_close_msec = 0;
+		return time_to_close_msec;
 	}
-	else if(column_ix == col_runnersNotFinished) {
```

Comment:
additionally `else` keywords removed to make the function consistent, they are not needed anyway